src/pages/user-profile/user-profile.tsx

```diff
@@ -0,0 +1,48 @@
+import { UserController } from 'networking/controllers/user-controller';
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import styles from './profile.module.scss';
+
+const UserProfile = () => {
+  type Params = {
+    id: string,
+  };
+  const { id } = useParams<Params>();
+  const [user, setUser] = useState<FullDummyUser>();
+
+  useEffect(() => {
+    UserController.getUser(id).then((value) => {
+      setUser(value);
+    });
+  }, [id]);
+
+  return (
+    <>
+      <h1>User profile 👤</h1>
+      {!user && (
+        <p>Loading...</p>
+      )}
+      {user && (
+        <div className={styles.userInfo}>
+          <div className={styles.profilePicture}>
+            <img alt="profile" src={user?.picture} />
+          </div>
+          <div className={styles.wrapper}>
```

Comment:
what do you think of adding some styles to the labels? Maybe just making it bold could look nice